site/es/blog/new-in-devtools-92/index.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: "layouts/blog-post.njk"
+title: "Qué hay de nuevo en DevTools (Chrome 92)"
+authors:
+  - jecelynyeen
+date: 2021-07-26
+updated: 2021-07-26
+description:
+  "Editor de CSS Grid, Soporte para la redeclaración de const en la consola, Visor de orden de los elementos y más."
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/XtJztwxzQqOWhOHrKmhM.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-92
+---
+
+*Traducción cedida por [Miguel Ángel](https://twitter.com/midudev).*
```

Comment:
After discuss with the team, we will cite the translation authors as "Translation provided by [author](link).". Let me know if the translation and your profile link is correct!